idd/Energy+.idd.in

```diff
@@ -50198,6 +50202,7 @@ HeatExchanger:AirToAir:FlatPlate,
       \type real
       \minimum> 0.0
       \units m3/s
+      \autosizable
```

Comment:
@Myoldmopar has a few loose bolts today.  Nice to be cloned - more productive that way.
 - A \default would be useful here
 - \min-fields needs to be increased to cover the new fields so their defaults will be filled in.